libs/vectorise/include/vectorise/meta/math_type_traits.hpp

```diff
@@ -90,8 +89,11 @@ using IfIsNotVectorRegister = fetch::meta::EnableIf<IsNotVectorRegister<T>, R>;
 template <typename DataType, typename ReturnType>
 using IfIsNotImplemented = fetch::meta::IfIsNotImplemented<DataType, ReturnType>;
 
-template <typename DataType>
-using IfIsUnsignedInteger = fetch::meta::IfIsUnsignedInteger<DataType>;
+// template <typename DataType>
```

Comment:
I've removed this in my PR subsequently - merge the PR in again to ensure this doesn't get added back in via auto merge